app/src/main/java/com/github/se/signify/ui/screens/profile/FriendsListScreen.kt

```diff
@@ -399,3 +447,58 @@ fun UserTextName(name: String) {
       fontWeight = FontWeight.Bold,
       modifier = Modifier.padding(start = 16.dp))
 }
+
+@Composable
+fun TextButton(onClickAction: () -> Unit, text: String, color: ButtonColors, testTag: String) {
```

Comment:
This function is the same as the `UtilTextButton()` in Utils.kt, so it is unnecessary. You can remove it.